src/platform_impl/windows/dark_mode.rs

```diff
@@ -79,10 +79,14 @@ pub fn allow_dark_mode_for_app(is_dark_mode: bool) {
   });
 
   if let Some(ver) = *WIN10_BUILD_VERSION {
+    // in case Windows 10 Build version < 18362, we have to call a special command (_allow_dark_mode_for_app)
     if ver < 18362 {
-      if let Some(_allow_dark_mode_for_app) = *ALLOW_DARK_MODE_FOR_APP {
-        unsafe { _allow_dark_mode_for_app(is_dark_mode) };
-      }
+        // but only call that command if the dark mode is supported...
+        if *DARK_MODE_SUPPORTED {
```

Comment:
we could wrap the whole function in `if *DARK_MODE_SUPPORTED` similar to how it is done in `try_window_theme` function